src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -49,12 +58,20 @@ string ArchiveNameForFile (string filename, string directoryPathInZip)
 			else {
 				pathName = Path.Combine (directoryPathInZip, Path.GetFileName (filename));
 			}
-			return pathName.Replace ("\\", "/");
+			return pathName.Replace ("\\", "/").TrimStart ('/');
+		}
+
+		void AddFileAndFlush (string filename, long fileLength, string archiveFileName, CompressionMethod compressionMethod)
+		{
+			filesWrittenTotalSize += fileLength;
+			zip.AddFile (filename, archiveFileName, compressionMethod: compressionMethod);
+			if ((filesWrittenTotalSize > ZipArchiveEx.ZipFlushSizeLimit || filesWrittenTotalCount > ZipArchiveEx.ZipFlushFilesLimit) && AutoFlush) {
```

Comment:
This is really minor, but the other places are doing `>=` and this does `>`.

Other places have:
```csharp
if (count >= ZipArchiveEx.ZipFlushFilesLimit) {
```